fixed everything ever oder so

RodiS40 opened the pull request:
ich hasse programmiere fr

*reminder wenn mer usem track gaht und zu schnell wieder ine gaht (ide app) crasht alles und ich ha kei lust das am 00:30 z fixe

guet nacht

RodiS40 committed:
updated esp code

RodiS40 committed:
updated esp32 code

added logging for GPS
gyro works without issues

RodiS40 committed:
Create info.txt

RodiS40 committed:
added track selector and track screen

RodiS40 committed:
added missing tracks and animations

RodiS40 committed:
added automatic track locator and fixed search

- when pressed it will ask for gps permissions and then auto locate where the user is using a boundingBox with min and max lat/lng
if a track is found it will show it, else will throw an error.
- fixed search algorithm by ditching everything and using regex like normal people

RodiS40 committed:
updated track screen

RodiS40 committed:
fixed everything ever

added telemetry with fallbacks
added connection to ESP
added stats table
fixed gauges
removed google maps for OSM
fixed map position and rotation
added locate me button to center the map

fick muetter es isch halbi 1

RodiS40 committed:
reworked to use BLE instead of WIFI

wifi didnt work dont ask

BLE sends data properly now
removed remaining GPS code
added fallback incase BLE is lost

*if you leave the track screen and go back in again too quick it will crash dont ask me why

RDLazic reviewed (approved):
Kseht guet us. HR approved

RodiS40 committed:
fixed crashing issues and added more features

fixed crashes
added proper dispose
added pitch roll
added logo to main menu
tried fixing 0-100 and didnt fix
adjusted main menu
adjusted dashboard tracks
improved ble latency
improved gps accuracy

RodiS40 committed:
Merge branch 'main' into cr7
